src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,46 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents an addition user command.
+ * Inherits from command class.
+ */
+public class AddCommand extends Command {
```

Comment:
Typically Class Names should be nouns, maybe change `AddCommand` to be `CommandAdder` instead? :)

Reply:
Noted! Will consider.

Reply:
same as Marcus's suggestion, nothing to add JYs!